roles/k3s/templates/neutron-calico-connect.j2

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
```

Comment:
the memories...

Reply:
technically calico seems to support floating IPs natively, but not in the operator deployed version. Could be interesting for future work.
https://projectcalico.docs.tigera.io/networking/add-floating-ip#enable-floating-ips